### DISCUSSION TITLE
Optimizer using only 1 CPU core

### DISCUSSION THREAD

**Discussion by ryzen3100** [Drakkar-Software/OctoBot#1682] [General]
Hi,
Is there a multi threading option for the Optimizer feature?
The process is using only 1 CPU core when optimizing, not utilizing the full performance potential (on both Debian and Windows, the latter is slower on the same system).

**Comment by Herklos** [comment]
Hi @ryzen3100,
There is no multi CPU support in OctoBot for now. It will only use 1 CPU to do all its computation. In the strategy optimizer we could actually use multiple CPUs to speed up the process. That's a good idea.
If you want to add it to OctoBot features roadmap please post your suggestion at https://feedback.octobot.online/.
